connector-common/src/main/java/com/redhat/cloud/notifications/connector/IncomingCloudEventFilter.java

```diff
@@ -18,7 +18,8 @@ public class IncomingCloudEventFilter implements Predicate {
 
     @Override
     public boolean matches(Exchange exchange) {
-        String connectorHeader = exchange.getIn().getHeader(X_RH_NOTIFICATIONS_CONNECTOR_HEADER, String.class);
-        return connectorConfig.getConnectorName().equals(connectorHeader);
+        return true;
+        //String connectorHeader = exchange.getIn().getHeader(X_RH_NOTIFICATIONS_CONNECTOR_HEADER, String.class);
+        //return connectorConfig.getConnectorName().equals(connectorHeader);
```

Comment:
I was wondering why one of the tests was failing. That's the reason 😄

Reply:
Sorry, yeah. Every time I test something I always forget to put everything back the way it was...

Reply:
No worries!